lib/AST/ASTImporter.cpp

```diff
@@ -6741,6 +6742,29 @@ Decl *ASTImporter::Import(Decl *FromD) {
   if (!FromD)
     return nullptr;
 
+  if (const auto *ND = dyn_cast<NamedDecl>(FromD))
```

Comment:
I would put this piece of code in a separate function.

Reply:
Done.